pkg/run/testdata/snapshots/env/tasks.toml

```diff
@@ -1,6 +1,11 @@
 [[task]]
   id = "test"
   type = "short"
-  cmd = "echo $VALUE"
-  env = { VALUE = "ENVIRONMENT VARIABLE" }
-
+  env = { SOME_ENV_VAR = "some env var value" }
+  cmd = """
+    echo "env variable set in task: "
+    echo "SOME_ENV_VAR: $SOME_ENV_VAR"
```

Comment:
I'd prefer more differentiation in the names for these two vars, along the lines of SET_IN_TASK and SET_IN_GO_TEST.